redash/handlers/query_results.py

```diff
@@ -262,10 +262,13 @@ def get(self, query_id=None, query_result_id=None, filetype='json'):
 
             filename = get_download_filename(query_result, query, filetype)
 
-            response.headers.add_header(
-                "Content-Disposition",
-                'attachment; filename="{}"'.format(filename.encode("utf-8"))
-            )
+            if filetype in ('csv', 'json') and request.args.get('download') in ['false', '0']:
+                response.headers.set('Content-Type', 'text/plain; charset=UTF-8')
```

Comment:
Can you please explain these changes (what are they needed for?). Thanks!

Reply:
This is for allowing users to click on the data source url that populated in Vega schema and review the data directly in browser without automatically downloading a file.

Reply:
I think in this case you have to set a proper MIME type for each extension. E.g. I have a chrome extensions that pretty-print both CSV and JSON (see screenshot with example), and both that extensions detect their targets by MIME type, so changes like this will break them.

![image](https://user-images.githubusercontent.com/12139186/60203651-c702c380-9855-11e9-97db-633f2dab80f2.png)

Reply:
I also have a JSON prettier extension installed, but couldn't find a good one for CSV. Which one are you using?

Since this change added a new URL parameter "download=false", it shouldn't break any existing links. I'm inclined to keep it. What do you think?